packages/waku/package.json

```diff
@@ -78,6 +78,7 @@
     "@vitejs/plugin-react": "4.3.1",
     "dotenv": "16.4.5",
     "hono": "4.4.11",
+    "react-server-action": "0.2.15",
```

Comment:
Despite the package name, it supports all RSC transforms, right?

Reply:
yes